chains/neutron/metadata.yaml

```diff
@@ -34,3 +34,14 @@ rpcUrls:
 slip44: 118
 transactionOverrides:
   gasPrice: "0.0075"
+supportedCurrencies:
```

Comment:
There's already a `nativeToken` field. I wonder if this should be folded into that? Or at least named `supportedTokens` or something or consistency?

Reply:
Or `supportedGasTokens`